sched/clock/clock_systime_ticks.c

```diff
@@ -90,7 +90,9 @@ clock_t clock_systime_ticks(void)
 
   up_timer_gettick(&ticks);
   return ticks;
-#elif defined(CONFIG_SCHED_TICKLESS)
+#elif defined(CONFIG_ALARM_ARCH) || \
+      defined(CONFIG_TIMER_ARCH) || \
+      defined(CONFIG_SCHED_TICKLESS)
```

Comment:
merge to line 88, and unify **up_timer_gettime/clock_time2ticks** to **up_timer_gettick**